src/screens/Products/ProductDetail.js

```diff
@@ -0,0 +1,266 @@
+  }
+
+  componentDidUpdate() {
+    LayoutAnimation.easeInEaseOut()
+  }
+
+  navigateToEditProduct() {
+    console.log("Button Pressed")
+  }
+
+
+  renderEditButton() {
+    if (this.state.editExpanded) {
+      return (
+        <CardSection style={{ backgroundColor: 'white' }}>
+          <Button
+            style={{ height: 20, backgroundColor: 'white' }}
+            title="Edit"
+            textColor="orange"
+            onPress={() => this.navigateToEditProduct()}
+          />
+          <Button
+            style={{ height: 25, backgroundColor: 'white' }}
+            title="Message"
+            textColor="orange"
+            onPress={() => console.log('Pressed message button')}
+          />
+        </CardSection>
+      )
+    }
+  }
+  render() {
+    return (
+      <Wallpaper style={styles.container}>
+        <ScrollView style={{ flex: 1.8 }}>
+          <Card>
+            <CardSection
+              style={{
+                backgroundColor: 'rgb(71, 77, 84)',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.headerText}>{this.props.Title}</Text>
+              <Icon
+                name="dots-vertical"
+                size={30}
+                color="orange"
+                type="material-community"
+                underlayColor={'rgb(71, 77, 84)'}
+                onPress={() =>
+                  this.setState({ editExpanded: !this.state.editExpanded })
+                }
+              />
+            </CardSection>
+            <CardSection>
+              <ProfilePicture
+                source={{ uri: this.props.mainPictureURL }}
+                style={styles.image}
+              />
+            </CardSection>
+            {this.renderEditButton()}
+          </Card>
+          <Card>
+            <CardSection
+              style={{
+                backgroundColor: 'rgb(71, 77, 84)',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.headerText}>{this.props.MainCategory}</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.descriptionStyle}>
+                {this.props.Description}
+              </Text>
+            </CardSection>
+          </Card>
+          <Card>
+            <CardSection
+              style={{
+                backgroundColor: 'rgb(71, 77, 84)',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.headerText}>Info</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.ProductInfo}>Sub-Cat</Text>
```

Comment:
Category titles should match the ones used in other components (like add product)